core/platform/auth/firebase_auth_services.py

```diff
@@ -462,10 +472,11 @@ def _get_auth_claims_from_session_cookie(cookie):
             with _firebase_admin_context():
                 return _create_auth_claims(
                     firebase_auth.verify_session_cookie(cookie))
+        # NOTE: We're OK with cookies becoming obsolete with time, and will
+        # suppress the errors related to this. See the constant for details.
```

Comment:
Try not to do this doc referencing -- docs change and references go stale. Just explain the reasoning here.